src/cascadia/TerminalControl/IControlSettings.idl

```diff
@@ -48,6 +49,7 @@ namespace Microsoft.Terminal.Control
         Microsoft.Terminal.Control.IKeyBindings KeyBindings { get; };
 
         Boolean CopyOnSelect { get; };
+        Microsoft.Terminal.Control.CopyFormat CopyFormatting { get; };

```

Comment:
I feel like `CopyFormat` sounds better than `CopyFormatting`.

Reply:
`CopyFormat` does sound better. Though, `CopyFormatting` is what we have used in global settings, and so far while bringing those settings into the `TerminalSettings` (to hand them over to other parts of the application), we have [preserved the names of the individual settings](https://github.com/microsoft/terminal/pull/16480/files#diff-c7a3e786ae62703d3746f9a1fc58704e0c3644d7816bf7596d1c17e045166677R354-R367) across global settings and `TerminalSettings`. Renaming `CopyFormatting` to `CopyFormat` would break that (maybe unintended) convention.

Is it okay to do this?

Reply:
Oh, nevermind. I wasn't aware that it was named `CopyFormatting` elsewhere already.